### ISSUE TITLE
API seems to cut out sometimes

### ISSUE THREAD

**Issue by jonpage88** [timcharper/homebridge-vivint#23]
Homebridge plugin seems to stop updating the contact sensors.  Contact senors will be updating as normal but then it will stop updating.  If I look in the Vivint app it is updated but homebridge does not match.  If I check the logs on homebridge I see the events have stopped coming in.  This will last for a little while then everything will stop working again.  While the contact senors are not updating I am still able to control locks are arm the panel as normal.

**Comment by daymondm** [comment]
yeah, I've noticed this as well. The API will refresh itself every so often (as defined in the config file) and catch any updates that the API doesn't send. You can lower that time to more quickly capture the updates, but it seems the API stream is fairly unreliable at times.

**Comment by balansse** [comment]
@jonpage88 check out my [fork](https://github.com/balansse/homebridge-vivint), in one of the recent updates I added changes to increase the reliability of the event stream, making device statuses update better.
